stdlib/testing/testdata/csvloop.flux

```diff
@@ -0,0 +1,27 @@
+import "csv"
+import "testing"
+
+// the 'long' that's given as a default table name was causing the decoder to loop infinitely.
+inData = "
+#datatype,string,long,string,long
+#group,false,false,true,false
+#default,_result,long,,
+,result,table,key,val
+,,0,k,0
+,,0,k,1
+,,0,k,2
+"
+
+// reading the data should result in a correctly read table that forgets about the default.
+outData = "
+#datatype,string,long,string,long
+#group,false,false,true,false
+#default,_result,,,
+,result,table,key,val
+,,0,k,0
+,,0,k,1
+,,0,k,2
+"
+
+

```

Comment:
It would be good to link to an issue here and note that this should actually be an error since `long` will not parse as an int, and the `#datatype` line says the tableID column should be a long.